src/components/implementation/no_interface/cFE_booter/.gitignore

```diff
@@ -0,0 +1,3 @@
+gen
```

Comment:
why another .gitignore in the same git repo?

Reply:
It's more flexible. We don't need super long paths in the top level `.gitignore`

Reply:
I see. I'm not sure how this works, does git ignore files from both local .gitignore and the global repository .gitignore? 
Also, this is probably the longest path we can have in our build system (in terms of levels, not names), so it's not super-long if you think of it.